engine/src/main/java/com/datatorrent/stram/StramClient.java

```diff
@@ -454,6 +456,7 @@ public void startApplication() throws YarnException, IOException
         appPath = new Path(configuredAppPath);
       }
       String libJarsCsv = copyFromLocal(fs, appPath, localJarFiles.toArray(new String[]{}));
+      processSSLConfig(dag.getValue(Context.DAGContext.SSL_CONFIG), fs, appPath, localResources);
```

Comment:
This is about setting up SSL resources if specified isn't it. How about calling it something like prepareSSLResources or setupSSLResources. Also, consider whether we need to breakout this into a separate method as this is not in line with the pattern in this method where other types of resources are being handled inline in the method.